### ISSUE TITLE
Create a classifier

### ISSUE THREAD

**Issue by rodrigoAMF** [rodrigoAMF/vehicle-classification#7]
Using PyTorch create a classifier to classify the images as one of the seven classes according to a softmax output.
